packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,352 @@
+      const {$ctx} = request;
+      ($ctx.request.getReq() as any).$ctx = undefined;
+      await $ctx.finish();
+    });
+
+    await rawApp.register(fastifyMiddie);
+
+    const plugins = await this.resolvePlugins();
+
+    for (const plugin of plugins) {
+      // console.log("register", (plugin.use as any)[Symbol.for("plugin-meta")].name, plugin.options)
+      await rawApp.register(plugin.use as any, plugin.options!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)